src/Components/Publishing/ToolTip/TooltipsDataLoader.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react"
+import { QueryRenderer, graphql } from "react-relay"
+import { createEnvironment } from "../../../Relay/createEnvironment"
+import { ArticleData } from "../Typings"
+import PropTypes from "prop-types"
+
+interface Props {
+  article: ArticleData
+}
+
+const environment = createEnvironment()
+
+export class TooltipsDataLoader extends Component<Props> {
+  render() {
+    return (
+      <QueryRenderer
+        environment={environment}
+        query={graphql`
+          query TooltipsDataLoaderQuery($artistIds: [String!]) {
+            artists(slugs: $artistIds) {
+              id
+              name
+              href
+              image {
+                url
+              }
+              bio
+            }
+          }
+        `}
+        variables={{
+          artistIds: ["andy-warhol", "pablo-picasso"],
```

Comment:
Need to add other entityTypes and replace these hardcoded values with a call to `getArtsySlugsFromArticle`